src/card/basic/card.directive.js

```diff
@@ -180,9 +180,17 @@ angular.module('patternfly.card').directive('pfCard', function () {
         return ($scope.headTitle || $scope.showFilterInHeader());
       };
 
+      $scope.getHeaderClass = function () {
+        return (!$scope.showTitlesSeparator || $scope.showTitlesSeparator === 'true' ? 'card-pf-heading' : 'card-pf-heading-no-bottom');
+      };
```

Comment:
That logic was in the html template, I moved into to the controller function per @erundle comment above :smile:  HTML is using ng-class